CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing
+
+Build the OCI image:
+
+```bash
+cd jenkins_agent_k8s_rock
+rockcraft
+```
+
+Push the OCI image to microk8s:
```

Comment:
Maybe add a note that the microk8s registry has to be enabled? In the [Indico Contribute docs](https://charmhub.io/indico/docs/how-to-contribute) its mentioned like this: 

>Note that the microk8s registry needs to be enabled using microk8s enable registry.

Reply:
Updated, thank you!